evennia/commands/default/muxcommand.py

```diff
@@ -109,15 +114,44 @@ def parse(self):
             else:
                 args = ""
                 switches = switches[0].split('/')
+            # Parse mux options, comparing them against user-provided switches, expanding abbreviations.
+            if hasattr(self, "options") and self.options and switches:
+                # If specific options are known, test them against given switches.
+                valid_switches, unused_switches, extra_switches = [], [], []
+                for element in switches:
+                    option_check = [each for each in self.options if each.lower().startswith(element.lower())]
```

Comment:
I think an exact match should be given priority here. That is, consider two switches `test` and `testingthings`. If the user enters `/test`, this system will find two `.startswith` matches and give an unambiguous error. So the exact match should be tested first, and only if that does not return any match should a fuzzy match be attempted. 
(In the command parser we also consider the length of the fuzzy-matches to score the results but that's a bit overkill for this case).

Reply:
This was easy to add.  I'll commit it soon.